### PR TITLE
Update iq.m3u

### PR DESCRIPTION
**Pull request by Redo-San**
I added more Iraqi channels, and I hope that the links will work outside Iraq❤🌹

### CODE REVIEW

File: streams/iq.m3u [hunk 1]
```diff
@@ -1,20 +1,72 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="12ArabiaTV.iq",12 Arabia TV (1080p)
+https://livestream.almanasa.tv:8029/news/ch19/ch19_1080.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:3]:
```suggestion
#EXTINF:-1 tvg-id="12ArabiaTV.iq",12 Arabia TV (1080p) [Geo-blocked]
https://livestream.almanasa.tv:8029/news/ch19/ch19_1080.m3u8
```

almanasa.tv doesn't work for me. If it works for you then please add `[Geo-blocked]` to every almanasa.tv link

File: streams/iq.m3u [hunk 2]
```diff
@@ -97,6 +163,12 @@ https://5ab29cc78f681.streamlock.net/PayamTV/livestream/playlist.m3u8
 https://cdn1.karwan.tv/spi-quran/index.m3u8
 #EXTINF:-1 tvg-id="RudawTV.iq",Rudaw TV (1080p)
 https://svs.itworkscdn.net/rudawlive/rudawlive.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="SafeerTV.iq",Safeer TV (1080p)
+https://livestream.almanasa.tv:8115/islamic/ch1/ch1_1080.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:167]:
```suggestion
#EXTINF:-1 tvg-id="SafeerTV.iq",Safeer TV (1080p)
https://cdn3.wowza.com/1/YStGZlJRdktzZkdK/VXROZWNW/hls/live/playlist.m3u8
#EXTINF:-1 tvg-id="SafeerTV.iq",Safeer TV (1080p)
https://livestream.almanasa.tv:8115/islamic/ch1/ch1_1080.m3u8
```

File: streams/iq.m3u (hunk 1 above)
Comment (Redo-San) [reply streams/iq.m3u:3]: Yes it works, I will add [Geo-blocked] to it.

File: streams/iq.m3u (hunk 2 above)
Comment (Redo-San) [reply streams/iq.m3u:171]: Oh sorry for that, I'll correct it now

**Comment by UltraHDR** [comment]
Thanks for your contribution!

**Review by UltraHDR — DISMISSED** [review]

**Review by UltraHDR — APPROVED** [review]

**Review by Carlinhos027 — APPROVED** [review]

**Pull request merged**